templates/admin/admin-classes.html

```diff
@@ -37,7 +37,7 @@ <h4 class="text-center">Total amount of shown classes: {{ classes|length }} <br>
             <td class="error_percentage_week_cell">{% if class.stats.week.runs > 0 %}{{(class.stats.week.fails / class.stats.week.runs * 100)|round(2)}}{% else %}0.00{% endif %}%</td>
             <td class="runs_cell">{{class.stats.total.runs}}</td>
             <td class="error_percentage_cell">{% if class.stats.total.runs > 0 %}{{(class.stats.total.fails / class.stats.total.runs * 100)|round(2)}}{% else %}0.00{% endif %}%</td>
-            <td id="statistics_cell"><a href="/stats/class/{{class.id}}" target="_blank">Click here</a></td>
+            <td id="statistics_cell"><a href="/live_stats/class/{{class.id}}" target="_blank">Click here</a></td>
```

Comment:
This is an error that was there before you edited this files, because I just tested it in main and it also failed, but this button made it apparent. When trying to access the class is returning this error:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2213, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2193, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/workspace/website/auth.py", line 213, in inner
    return f(*args, user=current_user(), **kws)
  File "/workspace/website/statistics.py", line 343, in render_live_stats
    adventures = self.get_class_live_stats(user, class_)
  File "/workspace/website/statistics.py", line 378, in get_class_live_stats
    adventures = self.__get_adventures_for_overview(user, class_['id'])
  File "/workspace/website/statistics.py", line 327, in __get_adventures_for_overview
    return _get_available_adventures(adventures, teacher_adventures, customizations, last_adventures)
  File "/workspace/website/statistics.py", line 1119, in _get_available_adventures
    adventure_name = adventure_names[adventure_key]
KeyError: '3f8aea42eb324f08a16776671498dd1b'
```

It doesn't happen when the user is `teacher1` so this is probably an error in the way we are retrieving the teacher adventures from the database. Would you like to tackle this in this PR? If not, there's no problem! I can also open a separate issue for this and work on it later.

Reply:
If we are cleaning up, we might as well also delete all code related to the classes button for the admin: 
<img width="186" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/78603b03-b093-4fec-8168-f338764cab36">

(in this PR or in another one)

Since there are soooo many classes in the database, this page often crashes so I never use it and look at my local copy of the database if I want to find info.

Reply:
Great, I fixed both the bug and removed the classes overview.